lib/omb-prompt-base.sh

```diff
@@ -341,15 +341,15 @@ function svn_prompt_vars {
 # - .hg is located in ~/Projects/Foo/.hg
 # - get_hg_root starts at ~/Projects/Foo/Bar and sees that there is no .hg directory, so then it goes into ~/Projects/Foo
 function get_hg_root {
-    local CURRENT_DIR=$(pwd)
+    local CURRENT_DIR="$PWD"
```

Comment:
Quoting the parameter expansions in the right-hand sides of variable assignments are actually not needed. I'd like to normalize this in the codebase at some point, but maybe it is good to remove the unnecessary quoting in this chance.
```suggestion
    local CURRENT_DIR=$PWD
```

Reply:
Thank you for the correction - I prefer the quote-less approach as well